kpi/forms.py

```diff
@@ -65,3 +74,31 @@ class Meta:
             # The 'password' field appears without adding it here; adding it
             # anyway results in a duplicate
         ]
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # Intentional _() call on dynamic string because the default choices
+        # are translated (see static_lists.py)
+        self.fields['sector'].choices = (('', ''),) + tuple(
+            (s, _(s)) for s in constance.config.SECTOR_CHOICES.split('\n')
+        )
+
+        # It's easier to _remove_ unwanted fields here in the constructor
+        # than to add a new fields *shrug*
+        desired_metadata_fields = json.loads(
+            constance.config.USER_METADATA_FIELDS
+        )
+        desired_metadata_fields = {
+            field['name']: field for field in desired_metadata_fields
+        }
+        for field_name in list(self.fields.keys()):
```

Comment:
For the record: `.keys()` is redundant but more readable (IMHO) 

```python
>>> d = {'foo':'bar', 'baz': 'qux'}
>>> list(d.keys())
['foo', 'baz']
>>> list(d)
['foo', 'baz']
```

Reply:
I'll confess: I didn't do it for readability, but rather because I forgot that `.keys()` wasn't necessary :blush: I do kind of like leaving it the way it is for explicitness, though.